.travis.yml

```diff
@@ -52,5 +52,4 @@ script:
       fi;
     - ./travis.sh
 after_failure:
-    - cat boost-maven/boost-maven-plugin/target/it/test-dev-release/build.log
     - cat boost-maven/boost-maven-plugin/target/it/test-spring-boot-docker/build.log
```

Comment:
Seems a bit surprising you'd stop 'cat'-ing the one you didn't fix?  Or can we just not cat both?

Reply:
Sorry not sure what you mean, `test-dev-release` is passing now, still trying to fix `test-spring-boot-docker` though. I'll remove all the `cat`s once I fix all the tests.

Reply:
I already am not sure why I said that :)  I think maybe because of the commit msg: `Fixing spring boot docker test` ?   Not a huge deal if so..